src/tensorlake/cli/_common.py

```diff
@@ -13,38 +13,53 @@
     VERSION = "unknown"
 
 
+from tensorlake.http_client import TensorlakeClient
+
+
 @dataclass
 class AuthContext:
-    """Class for CLI authentication context"""
+    """Class for CLI authentication context."""
 
     api_key: Optional[str] = None
+    base_url: str = None
     version: str = VERSION
     _client: Optional[httpx.Client] = None
     _introspect_response: Optional[httpx.Response] = None
+    _tensorlake_client: Optional[TensorlakeClient] = None
 
     def __post_init__(self):
         self.api_key = os.getenv("TENSORLAKE_API_KEY")
```

Comment:
The commands you add in this PR might be used by OSS Indexify cluster users. Might be a good idea to support a missing API key in case direct communication with Indexify cluster is needed for the user.